tests/request_helper_test.py

```diff
@@ -103,11 +103,20 @@ def test_is_tracing_returns_what_tracing_percent_method_returns_for_rest(
 
 
 def test_get_trace_id_returns_header_value_if_present(dummy_request):
-    dummy_request.headers = {'X-B3-TraceId': '17133d482ba4f605'}
+    dummy_request.headers = {'X-B3-TraceId': '48485a3953bb6124'}
```

Comment:
changed the existing test, as it more clearly shows the generator isn't being used